app/controllers/api/stories_controller.rb

```diff
@@ -7,7 +7,7 @@ class Api::StoriesController < Api::BaseController
 
   filter_resources_by :series_id, :account_id, :network_id
 
-  filter_params :highlighted, :purchased, :v4, :text
+  filter_params :highlighted, :purchased, :v4, :text, before: :time, after: :time
```

Comment:
I used `before` and `after` as the filter names - which is a bit purposefully-ambiguous.  Thinking that at some point we also might make this search `updated_at` for stories without a published/released date set.

But generally - I'm more okay with these simplistic names for a `?filter=<name>` rather than as a query param of their own.